test/tests/transfers/general_field/nearest_node/regular/main_array.i

```diff
@@ -97,8 +97,6 @@
     variable = 'from_sub from_sub'
     target_variable_components = '0 1'
     source_type = 'nodes nodes'
-    # to stabilize nearest-node results in parallel
-    search_value_conflicts = false
```

Comment:
Can you explain? The option is unstable in parallel? Is there something that still needs addressing?

Reply:
it was with the bug.
Turning the diagnostics back on is possible because the bug is removed